### DISCUSSION TITLE
CVE-2023-5720

### DISCUSSION THREAD

**Discussion by benedekh** [quarkusio/quarkus#37424] [Q&A] [answered]
Hello all,

is there any info about when [CVE-2023-5720](https://nvd.nist.gov/vuln/detail/CVE-2023-5720) will be fixed in Quarkus? It keeps showing up in our vulnerability reports.

**Comment by sberyozkin** [comment, marked as answer]
@benedekh Hi, it has been fixed in 3.2.8, we have sent updates to NVD to reflect it, hopefully it will help to avoid false positives

**Reply by sberyozkin** [reply]
@benedekh Status has been updated, https://nvd.nist.gov/vuln/detail/CVE-2023-5720, so hopefully there will be no more false positives related to it

**Reply by benedekh** [reply]
Thank you so much for the quick response, the FP findings have disappeared from the report.
